A.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="A.css">
+</head>
+<body>
+  <h1>Hello Microverse!</h1>
+</body>
+</html>
```

Comment:
- [OPTIONAL] Same here the commit message should be descriptive. For example, Add xxxx.html file. :grin: